packages/data-point/lib/entity-types/entity-entry/resolve.js

```diff
@@ -1,11 +1,15 @@
 const _ = require('lodash')
 const utils = require('../../utils')
 
+/**
+ * @param {Accumulator} acc
+ * @param {Function} resolveReducer
+ * @return {Promise}
+ */
 function resolve (acc, resolveReducer) {
   const contextTransform = acc.reducer.spec.value
-  let racc = acc
-  racc = utils.set(acc, 'value', _.defaultTo(acc.value, {}))
-  return resolveReducer(racc, contextTransform)
+  const racc = utils.set(acc, 'value', _.defaultTo(acc.value, {}))
+  return resolveReducer(racc, contextTransform, [['value']])
```

Comment:
These nested arrays are in a few places. I'm still not exactly sure why, tbh. What's the format that is produced and why is it being done that way? (Sorry if I'm overlooking something obvious.)

Reply:
The format can be seen by looking at the _stack strings in the snapshots.

Reply:
The values in the nested arrays will be printed with brackets and no preceding whitespace, so that we get things like `'entry:example[value]'`